File: src/main/java/ru/melonhell/bulletphysics/bullet/collision/space/cache/fluid/impl/EmptyFluidCache.java
package ru.melonhell.bulletphysics.bullet.collision.space.cache.fluid.impl;

import ru.melonhell.bulletphysics.bullet.collision.space.cache.fluid.FluidCache;
import ru.melonhell.bulletphysics.bullet.collision.space.cache.fluid.data.FluidColumn;
import ru.melonhell.bulletphysics.nms.wrappers.BlockPos;

public class EmptyFluidCache implements FluidCache {
    @Override
    public void refreshFluidData(BlockPos blockPos) {

    }

    @Override
    public FluidColumn getFluidColumn(BlockPos blockPos) {
        return null;
    }
}

File: src/main/java/ru/melonhell/bulletphysics/bullet/collision/space/cache/fluid/impl/EmptyFluidCache.kt
package ru.melonhell.bulletphysics.bullet.collision.space.cache.fluid.impl

import ru.melonhell.bulletphysics.bullet.collision.space.cache.fluid.FluidCache
import ru.melonhell.bulletphysics.bullet.collision.space.cache.fluid.data.FluidColumn
import ru.melonhell.bulletphysics.nms.wrappers.BlockPos

class EmptyFluidCache : FluidCache {
    override fun refreshFluidData(blockPos: BlockPos) = Unit
    override fun getFluidColumn(blockPos: BlockPos): FluidColumn? = null
}